sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ObjectExpressionsSuite.scala

```diff
@@ -360,8 +361,14 @@ class ObjectExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     stack.add(2)
     stack.add(3)
 
+    val itemFromScalaVersion = if (Utils.isScala212) {
+      (Seq(1, 2, 3), ObjectType(classOf[WrappedArray[Int]]))
+    } else {
+      (WrappedArray.make(Array(1, 2, 3)), ObjectType(classOf[WrappedArray[Int]]))
```

Comment:
`immutable.Seq` data can't convert to  `mutable.Seq` type, so this input is distinguished